README.md

```diff
@@ -1,17 +1,38 @@
-# Sample
+﻿# ABOUT
 
-This module is for fun and trying out new crazy and wacky things with the game. It houses a number of example blocks and UI screens.
+Sample.
 
-### Credits for images:
+This module is for fun and trying out new crazy and wacky things with the game. 
+It houses a number of example blocks and UI screens.
 
-WhiteStone: http://www.planetminecraft.com/texture_pack/new-start---by-loufisch/
+ 
+A person can test their UI screens and other various creations in this module.
+This module is open to local developments and tests.
+It is more inclined towards developers who want to test stuff; but players can enjoy it too!
+
+# LICENSE
+
+The module is licensed under Apache 2.0 license.
+
+# CREDITS
+
+Credits for images:
```

Comment:
You first have a credits heading, then a second line proclaiming `credits for ..` . Change the original "# CREDITS" to "# Credits for images", and remove this line.